apps/qubit/modules/default/actions/browseAction.class.php

```diff
@@ -209,11 +209,18 @@ public function execute($request)
       $skip = ($request->page - 1) * $this->limit;
     }
 
-    // Avoid pagination over 10000 records
-    if ((int)$this->limit + (int)$skip > 10000)
+    // Avoid pagination over ES' max result window config (default: 10000)
+    $maxResultWindow = arElasticSearchPluginConfiguration::getMaxResultWindow();
+
+    if ((int)$this->limit + (int)$skip > $maxResultWindow)
     {
       // Show alert
-      $message = $this->context->i18n->__("We've redirected you to the first page of results. To avoid using vast amounts of memory, AtoM limits pagination to 10,000 records. To view the last records in the current result set, try changing the sort direction.");
+      $message = $this->context->i18n->__(
+        "We've redirected you to the first page of results.
```

Comment:
Whaa! You can just wrap a PHP string over multiple lines like this? 🤯   I guess the linebreaks don't show up in HTML?

Reply:
Yeah, but I guess it may affect the translations, so I'll add the dots.